### PR TITLE
Replace try with safe operator 

### PR DESCRIPTION
**Pull request by sonalkr132**
Supersedes: #1863

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -11,6 +12,7 @@ AllCops:
   DisplayStyleGuide: true
   TargetRubyVersion: 2.3
 
+
```
Comment (simi) [inline .rubocop.yml]: :scissors: :question:

Comment (sonalkr132) [reply .rubocop.yml]: done

File: lib/safe_navigation_cop.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+module RuboCop::Cop
```
Comment (simi) [inline lib/safe_navigation_cop.rb]: Any reason to not use rubocop-rails' [SafeNavigation cop](https://github.com/rubocop-hq/rubocop-rails/blob/4712f515c67efc993ca8c6f62943bb761ff0b5df/lib/rubocop/cop/rails/safe_navigation.rb)?

Comment (sonalkr132) [reply lib/safe_navigation_cop.rb]: We don't have `rubocop-rails` in our Gemfile. We can add it someday. One would have to go through default checks and fix offences.

**Pull request merged**